src/BuildIntegration/BuildReadyToRunMultiModule.proj

```diff
@@ -0,0 +1,40 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <IlcCompileDependsOn>ComputeIlcCompileInputs;BuildOneReadyToRunMultiModule;AddModuleDependencies</IlcCompileDependsOn>
```

Comment:
I think this file should not make mention of "multimodule" since that's a specific CoreRT concept (and an old [CLR concept](https://blogs.msdn.microsoft.com/junfeng/2004/07/15/multimodule-assemblies/) to boot). I would name the project file something like `BuildReadyToRunAssembles.proj` and the targets something like `BuildAllReadyToRunAssemblies`, `BuildOneReadyToRunAssembly`.